tutorials/information_theoretic_acquisition_functions.ipynb

```diff
@@ -424,9 +426,8 @@
     "\n",
     "jes_lb = qLowerBoundJointEntropySearch(\n",
```

Comment:
```suggestion
    "jes_lb = qJointEntropySearch(\n",
```
I think a bunch of these need to be changed to fix the tutorial failure

Reply:
@esantorella Oh, right. Thanks, I'll go through that, too!

Reply:
@esantorella Done! Hopefully it's all good now =)